test/e2e/runner/setup.go

```diff
@@ -73,6 +73,10 @@ func Setup(testnet *e2e.Testnet, infp infra.Provider) error {
 			}
 		}
 
+		if testnet.Prometheus {
+			node.Prometheus = true
```

Comment:
Will this override the setting you introduced in `test/e2e/pkg/manifest.go` line 173 ?

Reply:
This code is no longer relevant, as I removed `node.Prometheus` flag (as explained above).